Makefile

```diff
@@ -2,15 +2,28 @@ UNAME:=$(shell uname)
 
 # Install settings
 PREFIX= /opt/cartesi
-BIN_INSTALL_PATH= $(PREFIX)/bin
-LIB_INSTALL_PATH= $(PREFIX)/lib
-SHARE_INSTALL_PATH= $(PREFIX)/share
+DESTDIR ?= $(PREFIX)
+MACHINE_EMULATOR_VERSION= 0.14.0
```

Comment:
I wish we avoid hard coding constants, otherwise we may have to keep updating this every release, what about leaving some unspecified default and let the caller decide this, like 0.0.0, or dev, or empty, or retrieve from git, not sure.

NOTE: recreated from a deletion mistake. (originally by: @edubart)